### ISSUE TITLE
Twitch Integrity Fail · Why some issues take forever to close

### ISSUE THREAD

**Issue by Ephellon** [Ephellon/Twitch-Tools#30]
Hello all,

**TL;DR** — Twitch's bot detection erroneously disables my account for several weeks at a time, please be patient with open issues.

----

I am unfortunately in a region (Mid-Western, United States) that Twitch deems "bot rich," and regularly get a [**Twitch Integrity Failure**](https://www.reddit.com/r/Twitch/comments/y8pwf6/i_cant_sign_into_my_account_i_get_an_error_every/) (Reddit) error for several weeks at a time. When I am able, I try to fix as many issues as I can.

When I do have the error, I **cannot** do the following:

- View or change some personal settings
- View, send, delete, or report whispers
- View notifications
- Buy or spend bits
- Collect or spend channel points
- Follow or unfollow channels
- Subscribe or unsubscribe to channels
- Report channels
- Report messages
- Moderate any channel (including my own)
- Start or stop raids
- View some ads
    - 😂 this one is especially funny because a different ad will play when I'm not in Twitch bot jail
    - Yes, I get the same exact ad for weeks on end... Which is why there's some ad-detection in this extension
- Log in or out of Twitch
- Access Twitch APIs
- *And much more...*
